wagtail/admin/views/account.py

```diff
@@ -162,13 +161,8 @@ class NotificationsSettingsPanel(BaseSettingsPanel):
     form_object = "profile"
 
     def is_active(self):
-        # Hide the panel if the user can't edit or publish pages
-        user_perms = UserPagePermissionsProxy(self.request.user)
-        if not user_perms.can_edit_pages() and not user_perms.can_publish_pages():
-            return False
```

Comment:
The checks for this is already handled in the form, which will delete the fields accordingly.

Also, this check seems a bit strange: it means that if we have other fields in `NotificationPreferencesForm` that should be shown regardless of these two conditions, they wouldn't show up if the user doesn't pass this check. Hence, I removed this.

Reply:
Yep - that does indeed look suspect!